client/my-sites/controller.js

```diff
@@ -162,13 +161,11 @@ module.exports = {
 		if ( ! siteID ) {
 			sites.selectAll();
 			context.store.dispatch( setAllSitesSelected() );
-			siteStatsStickyTabActions.saveFilterAndSlug( false, '' );
```

Comment:
If we wanted to populate a redux value, this is where we'd dispatch an action to do so. For now, 💥.